test/partially_blindrsa.test.ts

```diff
@@ -143,8 +126,12 @@ describe.each(vectors)('Errors-vec$#', (v: Vector) => {
 
 test.each(vectors)('TestVector_$#/safePrimes', (v: Vector) => {
     // It requires to seed the internal random number generator.
-    while (sjcl.random.isReady(undefined) === 0) {
-        sjcl.random.addEntropy(crypto.getRandomValues(new Uint32Array(4)), 128, undefined);
+    while (!sjcl.random.isReady(undefined)) {
+        sjcl.random.addEntropy(
+            Array.from(crypto.getRandomValues(new Uint32Array(4))),
+            128,
+            'undefined',
```

Comment:
should this be a constant? having a string `"undefined"` feels strange without any comment https://github.com/bitwiseshiftleft/sjcl/blob/master/core/random.js/#L117-L123

Reply:
moved this step into a single function.